src/locales/ko/mystery-encounters/teleporting-hijinks-dialogue.json

```
@@ -1,27 +1,27 @@
 {
-  "intro": "It's a strange machine, whirring noisily...",
-  "title": "Teleportating Hijinks",
-  "description": "The machine has a sign on it that reads:\n  \"To use, insert money then step into the capsule.\"\n\nPerhaps it can transport you somewhere...",
-  "query": "What will you do?",
+  "intro": "시끄럽게 윙윙 소리가 나는 이상한 기계를 발견했다….",
+  "title": "텔레포트 대소동",
+  "description": "기계에는 다음과 같은 표지판이 있습니다:\n  \"사용하려면, 돈을 넣고 캡슐에 들어가세요.\"\n\n어딘가로 당신을 데려다줄 수 있을겁니다…….",
+  "query": "어떻게 하시겠습니까?",
   "option": {
     "1": {
-      "label": "Put Money In",
-      "tooltip": "(-) Pay {{price, money}}\n(?) Teleport to New Biome",
-      "selected": "You insert some money, and the capsule opens.\nYou step inside..."
+      "label": "돈을 넣는다",
+      "tooltip": "(-) {{price, money}} 지불\n(?) 다른 바이옴으로 이동",
+      "selected": "돈을 넣자, 캡슐이 열렸다.\n당신은 안으로 들어갔다……."
     },
     "2": {
-      "label": "A Pokémon Helps",
-      "tooltip": "(-) {{option2PrimaryName}} Helps\n(+) {{option2PrimaryName}} gains EXP\n(?) Teleport to New Biome",
-      "disabled_tooltip": "You need a Steel or Electric Type Pokémon to choose this",
-      "selected": "{{option2PrimaryName}}'s Type allows it to bypass the machine's paywall!$The capsule opens, and you step inside..."
+      "label": "포켓몬의 도움받기",
```

```suggestion
      "label": "포켓몬의 도움",
```
깔끔하게 그냥 '도움'이라고만 하는 게 더 좋지 않을까 제안합니다.